src/coreclr/System.Private.CoreLib/src/System/Runtime/DependentHandle.cs

```diff
@@ -200,7 +200,7 @@ public DependentHandle(object? target, object? dependent)
         /// Sets the dependent object instance for the current handle to <see langword="null"/>.
         /// </summary>
         /// <remarks>This method mirrors the <see cref="Target"/> setter, but without allocation and input checks.</remarks>
-        internal void UnsafeSetTargetToNull()
+        internal readonly void UnsafeSetTargetToNull()
```

Comment:
Marking a setter `readonly` is a busted mechanic. I don't see how this is correct.